acsylla/factories.py

```diff
@@ -117,6 +117,9 @@ def create_statement(
     If `consistency` is provided, this will override the consistency value provided during the cluster
     creation.
     """
+    if statement.strip().lower().startswith("use"):
+        raise ValueError("For change keyspace use await session.set_keyspace(name)")
```

Comment:
From my understanding there is no side effect if the user makes this "by hand" instead of using new method proposed [1].

If not, i would not add this safe guard here (I would remove too for the the prepared one) since this will need to be executed every time that a new statement is created and will hurt for sure the performance.

WDYT?

[1] https://github.com/acsylla/acsylla/pull/64/files#diff-868c624f52f869a30fd8b9c40d43bdffa867165a7e292eb6c0ddb34ecd598d29R69

Reply:
I agree it's not very good for performance, but good for userfrendly. I'm for performance! 
But using the driver you need to understand what you are doing.

If user change keyspace over "USE" statement, C++ driver have not functions for give current keyspace(
May be I wrong, but I can't find this in docs and in code.

I want removing this, but may be we wrap this check in a special flag (for cqlsh for example)?